nucypher/config/constants.py

```diff
@@ -5,14 +5,22 @@
 
 import nucypher
 
+PACKAGE_NAME = 'nucypher'
+
 # Base Filepaths
 BASE_DIR = abspath(dirname(dirname(nucypher.__file__)))
 PROJECT_ROOT = abspath(dirname(nucypher.__file__))
-APP_DIR = AppDirs("nucypher", "NuCypher")
+APP_DIR = AppDirs(PACKAGE_NAME, "NuCypher")
 DEFAULT_CONFIG_ROOT = APP_DIR.user_data_dir
 
-#
-# seednodes
-#
+# Static Seednodes
 SeednodeMetadata = namedtuple('seednode', ['checksum_address', 'rest_host', 'rest_port'])
 SEEDNODES = tuple()
+
+# Sentry (Set to False to disable sending Errors and Logs to NuCypher's Sentry.)
+REPORT_TO_SENTRY = True
```

Comment:
Not necessarily related to this PR: Say I'm working with the CLI/demo script/etc. (i.e., anything that is not running with pytest) in my local environment, and I don't want to report to Sentry. Is this the only way to turn it off?

Reply:
Exactly. An issue here is that `nucypher` currently is the main application for Ursulas, but also  the library for all the remaining actors in NuCypher's narrative. Both will have different defaults.

Reply:
PR #526 is a reasonable fix for this - scoping sentry reports *only* to the CLI